app/controllers/api/v1/carts_controller.rb

```diff
@@ -16,7 +14,7 @@ def show
   end
 
   def create
-    cart = @user.carts.create!(cart_params)
+    cart = current_user.carts.create!(cart_params)
```

Comment:
I don't remember but in my mind, you`re able to use 
```ruby
Cart.create!(current_user)
```

Reply:
I also remembered this point, but it doesn't work now
![image](https://github.com/oleksiimikhno/book-store/assets/41300309/a64443f6-f316-49c1-aaa2-3d8e646e2c2e)